exoctk/utils.py

```diff
@@ -42,28 +42,28 @@
             '"ExoCTK Data Download" button on the ExoCTK website, or by using '
             'the exoctk.utils.download_exoctk_data() function.'
         )
-
-    # If the variable exists but doesn't point to a real location
-    if not os.path.exists(EXOCTK_DATA):
-        print(
-            'WARNING: The $EXOCTK_DATA environment variable is set to a location that '
-            'cannot be accessed.')
-
-    # If the variable exists, points to a real location, but is missing contents
-    for item in ['exoctk_contam', 'exoctk_log', 'fortney', 'generic', 'groups_integrations', 'modelgrid']:
-        if item not in [os.path.basename(item) for item in glob.glob(os.path.join(EXOCTK_DATA, '*'))]:
+    else:
+        # If the variable exists but doesn't point to a real location
+        if not os.path.exists(EXOCTK_DATA):
             print(
-                'WARNING: Missing {}/ directory from {}. Please ensure that the ExoCTK data package has been '
-                'downloaded. Users may retrieve this package by clicking the "ExoCTK Data Donwload" '
-                'button on the ExoCTK website, or by using the exoctk.utils.download_exoctk_data() '
-                'function'.format(item, EXOCTK_DATA))
-
-EXOCTK_CONTAM_DIR = os.path.join(EXOCTK_DATA, 'exoctk_contam/')
-EXOCTKLOG_DIR = os.path.join(EXOCTK_DATA, 'exoctk_log/')
-FORTGRID_DIR = os.path.join(EXOCTK_DATA, 'fortney/')
-GENERICGRID_DIR = os.path.join(EXOCTK_DATA, 'generic/')
-GROUPS_INTEGRATIONS_DIR = os.path.join(EXOCTK_DATA, 'groups_integrations/')
-MODELGRID_DIR = os.path.join(EXOCTK_DATA, 'modelgrid/')
+                'WARNING: The $EXOCTK_DATA environment variable is set to a location that '
+                'cannot be accessed.')
+
+        # If the variable exists, points to a real location, but is missing contents
+        for item in ['exoctk_contam', 'exoctk_log', 'fortney', 'generic', 'groups_integrations', 'modelgrid']:
+            if item not in [os.path.basename(item) for item in glob.glob(os.path.join(EXOCTK_DATA, '*'))]:
+                print(
+                    'WARNING: Missing {}/ directory from {}. Please ensure that the ExoCTK data package has been '
+                    'downloaded. Users may retrieve this package by clicking the "ExoCTK Data Donwload" '
```

Comment:
Typo: "Donwload" -> "Download"

Reply:
Thanks for catching this! Fixed it: https://github.com/ExoCTK/exoctk/pull/353/commits/a0e32692f46151220c05e9ed281039f43d4cf2f7